subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -185,3 +185,33 @@ Unlike the methods on the link:{groovyDslPath}/org.gradle.api.tasks.testing.Test
 An example of this behavior is shown in the first configuration example above, <<#sec:configuring_the_built_in_test_suite,Configuring the built-in `test` suite>>.
 
 Note that unlike the link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html[Test] task, the aforementioned methods on link:{groovyDslPath}/org.gradle.api.plugins.jvm.JvmTestSuite.html[JvmTestSuite] do not accept a closure or action for configuring the framework.  These framework configuration options can be set on the individual targets.
+
+== Outgoing Variants
+Each test suite creates an outgoing variant containing its test execution results.  These variants are designed for consumption by the <<test_report_aggregation_plugin#test_report_aggregation_plugin, Test Report Aggregation Plugin>>.
+
+The attributes will resemble the following.  User-configurable attributes are highlighted below the sample.
+
+.outgoingVariants task output
+[source,text]
+----
+--------------------------------------------------
+Variant testResultsElementsForTest
```

Comment:
This will now show up with an "(i)" indicator next to the name, because it is incubating.

Reply:
Nice - I will adjust the text.

Reply:
I added the `Description` too.  Very nice.